### DISCUSSION TITLE
GitHub Hosted Runners for Government

### DISCUSSION THREAD

**Discussion by michael-pang** [community/community#32246] [Actions]
I was unable to find this anywhere in the discussions or on the roadmap but I was curious if there are any plans for GitHub to seek FedRAMP certification for GitHub Hosted Runners? 

I ask because we are currently evaluating the usage of GitHub Hosted Runners and Self-Hosted Runners and are wary about our commercial deployments leveraging the GitHub Hosted Runners and then funneling Government deployments through Self-Hosted Runners that may behave differently at deployment time.

**Comment by 3kh0** [comment]
I do not think that there are plans for FedRAMP to be added.

*If this helped please mark this as the answer*
